pages/_document.tsx

```diff
@@ -87,6 +87,15 @@ class MyDocument extends Document {
         <body>
           <Main />
           <NextScript />
+
```

Comment:
I must admit I'm confused with those lines. Where do you configure the analytics?